### ISSUE TITLE
kinetics.A.getConversionFactorFromSItoCM() may be buggy; gives AssertionError

### ISSUE THREAD

**Issue by mazeau** [cfgoldsmith/RMG-Py#45]
I'm trying to rerun linear scaling, and I'm now getting an assertion error I believe is linked to us changing thermo 

```
AssertionError: 
              Gas phase reaction 
H(12)+C3H7(31)=C3H8(11)                             
              with kinetics 
Arrhenius(A=(3.48677e-12,'cm^3/(molecule*s)'), n=0.6, Ea=(-3.3472,'kJ/mol'), T0=(1,'K'), comment="""From training reaction 18 used for C_rad/H2/Cs;H_rad
Exact match found for rate rule [C_rad/H2/Cs;H_rad]
Euclidian distance = 0
family: R_Recombination""")
              with 2 reactants was expected to have
              kinetics.A.getConversionFactorFromSItoCM() = 1000000.0
              but instead it is 1.66053878316e-18
```

**Comment by rwest** [comment]
I think it's a units conversion error
1e6 / 1.66053878316e-18 = 6.0221418e+23  (Avogadro's number)
because the gas phase reaction is specified with Arrhenius kinetics in `'cm^3/(molecule*s)'` and not `'cm^3/(mole*s)'`. 
We could look at the training reaction 18 in the R_Recombination family, and maybe get rid of the problem that way.

But we should put also some more unit tests on `getConversionFactorFromSItoCM()` (which I think is in `quantities.py`) to reveal this bug (and then fix it).

OR maybe actually it's doing exactly the right thing, and the assertion statement was naive, and it shouldn't expect the answer to be `1.0e6 ** (numReactants - 1)`. 
Come to think of it, I think that might be it.

**Comment by rwest** [comment]
For now, you could just comment out the assertion error. Or turn it into an `if` block that logs a warning. 

But before we close this issue I'd like to know what's special about that reaction, and how it shows up when it ends up in a model on the Master branch.

**Comment by mazeau** [comment]
So I commented it out andmake cleaned and recompiled and now I have 
```
/Users/emilymazeau/Code/RMG-Py/rmgpy/rmg/main.py:984: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  p_min=self.Pmin, p_max=self.Pmax)
Traceback (most recent call last):
  File "/Users/emilymazeau/Code/RMG-Py/rmg.py", line 173, in <module>
    main()
  File "/Users/emilymazeau/Code/RMG-Py/rmg.py", line 167, in main
    rmg.execute(**kwargs)
  File "/Users/emilymazeau/Code/RMG-Py/rmgpy/rmg/main.py", line 940, in execute
    self.check_model()
  File "/Users/emilymazeau/Code/RMG-Py/rmgpy/rmg/main.py", line 984, in check_model
    p_min=self.Pmin, p_max=self.Pmax)
  File "rmgpy/reaction.py", line 1249, in rmgpy.reaction.Reaction.check_collision_limit_violation
  File "rmgpy/reaction.py", line 1271, in rmgpy.reaction.Reaction.check_collision_limit_violation
  File "rmgpy/reaction.py", line 1297, in rmgpy.reaction.Reaction.calculate_coll_limit
  File "rmgpy/reaction.py", line 1297, in rmgpy.reaction.Reaction.calculate_coll_limit
  File "rmgpy/reaction.py", line 1303, in rmgpy.reaction.Reaction.calculate_coll_limit
  File "rmgpy/reaction.py", line 1323, in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon
  File "rmgpy/reaction.py", line 1323, in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon
  File "rmgpy/reaction.py", line 1340, in rmgpy.reaction.Reaction.get_mean_sigma_and_epsilon
AttributeError: 'NoneType' object has no attribute 'sigma'
```

**Comment by rwest** [comment]
I think that's sufficiently different it should be a new issue

**Comment by rwest** [comment]
Probably fixed in f20599f513f2cd79672f287303b7733a75c79fe0
